### PR TITLE
treewide: cleanup old throw aliases

### PR DESCRIPTION
**Pull request by fabianhjr**
###### Description of changes

Delete throws older than 2020-01-01

###### Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.05 Release Notes (or backporting 21.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2205-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/desktops/xfce/default.nix [hunk 1]
```diff
@@ -169,8 +169,6 @@ lib.makeScope pkgs.newScope (self: with self; {
   xfce4_power_manager = xfce4-power-manager;
   xfce4_appfinder = xfce4-appfinder;
   xfce4_dev_tools = xfce4-dev-tools;
-  xfce4mixer = xfce4-mixer;
-  xfce4mixer_pulse = xfce4-mixer-pulse;
```
Comment (pennae) [inline pkgs/desktops/xfce/default.nix:173]: these (and more) don't throw, did you mean to remove those here as well?

Comment (fabianhjr) [reply pkgs/desktops/xfce/default.nix:173]: Yes, they did throw since they pointed to a throw alias

Comment (fabianhjr) [reply pkgs/desktops/xfce/default.nix:173]: Check lines 207 and 220 of the same file.

**Comment by pennae** [comment]
checked that the removed aliases are indeed as old as they claim. :+1:

**Comment by SuperSandro2000** [comment]
@fabianhjr please rebase and give me a ping to merge.

**Comment by fabianhjr** [comment]
Hi @SuperSandro2000, the conflict has been resolved and this PR has been rebased.

**Comment by SuperSandro2000** [comment]
@jonringer anything we need to take care of before?

**Comment by Artturin** [comment]
added an only-throws option to the remove-old-aliases script https://github.com/NixOS/nixpkgs/pull/165971

**Comment by jonringer** [comment]
> @jonringer anything we need to take care of before?

nope, this looks good to me

**Review by jonringer — APPROVED** [review]

**Pull request merged**